src/rm_spaces.py

```diff
@@ -0,0 +1,6 @@
+def rm_spaces(str):
```

Comment:
'str' is a built in function https://docs.python.org/3/library/functions.html you shouldn't override these functions.